README.md

```diff
@@ -176,6 +177,65 @@ interface Root extends Node {
 
 **Root** can be used as the _[root][term-root]_ of a _[tree][term-tree]_.
 
+### `Topper`
+
+```ts
+type TopperType =
+| 'DeepPortraitTopper'
+| 'DeepLandscapeTopper'
+| 'SplitTextTopper'
+| 'FullBleedTopper'
+| 'PodcastTopper'
+| 'OpinionTopper'
+| 'BrandedTopper'
+| 'BasicTopper'
+| 'TopperWithFlourish'
+| 'PartnerContentTopper'
+```
+
+```ts
+interface Topper extends Parent {
+	type: 'topper'
+	topperType: TopperType
+	backgroundColor: string // maybe a type? is this external??
+	children: [Headline, Intro, TopperVisual?]
```

Comment:
should these be keys rather than children? allows for further growth?

Reply:
i think it does make sense. 

the benefit of having them as children i guess would be if there wasn't a regimented structure, and the FE could receive/render _anything_ (like we do with `RichText`).

this probably isn't true for the topper